### ISSUE TITLE
Reproducible behavior based on pseudo-random seeds

### ISSUE THREAD

**Issue by schrum2** [schrum2/EvoCraft-SCOPE#21]
Although we want to see lots of random variation, we also need to be able to reproduce our "random" results if needed. Everything is pseudo-random, and if we use the same random seed and make the same decisions we should be able to reproduce the same results.

Add a command line parameter called RANDOM_SEED with a default value of 0. In `main.py` do the following:
```
import random
random.seed(args.RANDOM_SEED)
```
I believe that this should make all subsequence uses of randomness be consistent, but it sort of depends on how random number generators are used in various places in the code, so I'm not sure.

Basically, every time you initialize an evolution run with the same seed, you should see the exact same shapes. Verify that this is true. If it is, then you can then properly test #14

**Comment by alejmedinajr** [comment]
Added the command line parameter RANDOM_SEED and used it to test issue #14. When the default value for this command line parameter was set to a constant value, the same initial shapes were generated. However the latest commit now has the default as a random number within the range of 0-100.
